templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -736,7 +736,7 @@ Resources:
                     # Ensure awscli is up to date
                     yum install -y awscli jq
                     key_val=$(aws --region=${AWS::Region} ssm get-parameters --names "$key_name" --with-decryption | jq --raw-output '.Parameters[0] .Value')
-                    echo -e $key_val > $key_location
+                    echo -e "$key_val" > $key_location
```

Comment:
What's the difference between `$key_val` and `"$key_val"`? 🤔

Reply:
If the SSH key contains some special characters (which is probably the case for https://bitbucket.org/atlassian/atlassian-aws-deployment/issues/40/jira-quickstart-template-ssh-escaping) the command might not be executed correctly. When I think about that, I should probably use `'$key_val'`.

Reply:
Sounds good to me 😌